app/controllers/concerns/idv_step_concern.rb

```diff
@@ -50,9 +50,12 @@ def check_for_mail_only_outage
   end
 
   def redirect_for_mail_only
-    return redirect_to vendor_outage_url unless FeatureManagement.gpo_verification_enabled?
-
-    redirect_to idv_mail_only_warning_url
+    policy = Idv::GpoVerifyByMailPolicy.new(current_user, resolved_authn_context_result)
```

Comment:
Seems like this is constructed in many separate controllers. Would it make sense to have one policy on a mixin shared by all these controllers? Or just on ApplicationController?

Reply:
Yeah, when I was putting this together I was thinking that a mixin may make this a little cleaner.